docs/knowledgebase/integrate/subkey.md

```diff
@@ -184,7 +184,7 @@ Secret phrase `spend report solution aspect tilt omit market cancel what type ca
   Secret seed:      0x554b6fc625fbea8f56eb56262d92ccb083fd6eaaf5ee9a966eaab4db2062f4d0
   Public key (hex): 0x143fa4ecea108937a2324d36ee4cbce3c6f3a08b0499b276cd7adb7a7631a559
   Account ID:       0x143fa4ecea108937a2324d36ee4cbce3c6f3a08b0499b276cd7adb7a7631a559
-  SS58 Address:     5CXFinBHRrArHzmC6iYVHSSgY1wMQEdL2AiL6RmSEsFvWezd
+  SS58 Address:     D2sP6XA4DBn3eadsRMYBPoggcDbCuSWUYZ5V63PifURFkcm
```

Comment:
I am still not convinced this is the change we want to make at this time. I think we're still working to pin down the source of the unexpected behavior.

Reply:
In this case the example is using the `--network kusama` so if the `5CXF...` address actually comes out then this tag is broken and we shouldn't be recommending to install it at all.